### PR TITLE
fix #4425: local app should be downloadable

### LINKED ISSUE #4425: [local app] set proper content type for downloadable
### Bug description

Otherwise browser cannot download then:
Screenshot 2021-06-07 at 15 45 46

### Steps to reproduce

See above.

### Expected behavior

_No response_

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by akosyakov**

### CODE REVIEW

**Comment by akosyakov** [comment]
I have not tried yet, not sure how caddy matching works.

**Review by aledbf — APPROVED** [review]
LGTM

**Comment by aledbf** [comment]
```console
curl https://akosyakov-local-app-set-proper-4425.staging.gitpod-dev.com/static/bin/gitpod-local-companion-darwin -v
.....
< accept-ranges: bytes
< cache-control: public, max-age=31536000
< content-disposition: attachment
< content-security-policy: frame-ancestors self https://*.akosyakov-local-app-set-proper-4425.staging.gitpod-dev.com https://akosyakov-local-app-set-proper-4425.staging.gitpod-dev.com
< date: Tue, 08 Jun 2021 11:06:10 GMT
< etag: "qu0hdpaf9k0"
< referrer-policy: no-referrer-when-downgrade
< strict-transport-security: max-age=31536000
< vary: Accept-Encoding
< x-content-type-options: nosniff
< x-xss-protection: 1; mode=block
< content-type: application/octet-stream
< 
Warning: Binary output can mess up your terminal. Use "--output -" to tell 
Warning: curl to output it to your terminal anyway, or consider "--output 
Warning: <FILE>" to save to a file.
* Failure writing output to destination
* stopped the pause stream!
* Connection #0 to host akosyakov-local-app-set-proper-4425.staging.gitpod-dev.com left intact
```

**Comment by akosyakov** [comment]
hm, it downloads but uses the wrong name, namely gitpod-local-companion-linux.txt insted of gitpod-local-companion-linux

File: components/dashboard/conf/Caddyfile [hunk 1]
```diff
@@ -13,12 +12,21 @@
...
+
 	# static assets configure cache headers and do not check for changes
 	header @static_asset {
 		Cache-Control "public, max-age=31536000"
 		# remove Last-Modified header
 		-Last-Modified
 	}
+
+	header @bin_asset {
+		Content-Disposition: attachment
```
Comment (aledbf) [inline components/dashboard/conf/Caddyfile:28]: @akosyakov please add a line with `Content-Type application/octet-stream`

Comment (aledbf) [reply components/dashboard/conf/Caddyfile:28]: Also, remove the char `:`

**Comment by akosyakov** [comment]
Cool! It worked now. Thanks a log @aledbf!

**Pull request merged**